catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -391,11 +391,12 @@ public Metacard transform(InputStream input, String id)
                 metadata, id, metadataText, metacardType, useResourceTitleAsTitle);
         if (StringUtils.isNotBlank(bodyText)) {
           metacard.setAttribute(new AttributeImpl(Extracted.EXTRACTED_TEXT, bodyText));
+          processContentMetadataExtractors(bodyText, metacard);
```

Comment:
Same question for this.

Reply:
The interface doesn't guarantee that they have to handle a blank/null case. If it did, we could remove these checks.